lib/__tests__/test.js

```diff
@@ -0,0 +1,4 @@
+jest.autoMockOff();
+const runTestFixtures = require('../runTestFixtures.js');
```

Comment:
Very nice! However I think there's not much point in separating this into a runTestFixtures.js file. You can just keep the code here; after all, that code is entirely concerned with running the tests.